notebooks/7-rotation-speeds/interm_rotation_rate.ipynb

```diff
@@ -0,0 +1,2926 @@
+{
```

Comment:
<p>Perhaps we need to constrain our periodograms a bit here.</p><ul><li>Max period for a sector would need to be 14 days, as you've done</li><li>The min period would be determined by </li><li class="ql-indent-1">the TESS sample rate, which varies based on sector but is on the order of 20s for mission targets</li><li class="ql-indent-1">we may be able to rule out some of the ultra-low periods as oscillation rates rather than rotation rates. What is a "typical" rotation rate for these types of stars?</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='65'/>